doc/en/components/charts/chart-type-overview.md

```diff
@@ -1,32 +1,212 @@
+
+### Polar Area chart 
+
+The $PlatformShort$ Polar Area Chart or Polar Graph belongs to a group of polar charts and has a shape of a filled polygon which vertices or corners are located at the polar (angle/radius) coordinates of data points. The Polar Area Chart uses the same concepts of data plotting as the Scatter Chart but wraps data points around a circle rather than stretching them along a horizontal line. Like with other series types, multiple Polar Area Charts can be plotted in the same data chart and they can be overlaid on each other to show differences and similarities between data sets. Learn more about our [polar charts](chart-types-polar.md).
+
+### Scatter chart 
+
+The $PlatformShort$ Scatter Chart, or Scatter Graph, is used to show the relationship between two values using a Cartesian (X, Y) coordinate system to plot data. Each data point is rendered as the intersecting point of the data value on the X and Y Axis. Scatter charts draw attention to uneven intervals or clusters of data. They can highlight the deviation of collected data from predicted results and they are often used to plot scientific and statistical data. The $PlatformShort$ Scatter chart organizes and plots data chronologically (even if the data is not in chronological order before binding) on X-Axis and Y-Axis. Learn more about our [scatter charts](chart-types-scatter.md).
+
+### Spline Chart 
+
+The $PlatformShort$ Spline Chart, or Spline Graph is a type of category line graph shows the continuous data values represented by points connected by smooth line segments of one or more quantities over a period time for showing trends and performing comparative analysis. The Y-Axis (labels on left side) show a numeric value, while the X-Axis (bottom labels) are showing a time-series or comparison category. You can include one or more data sets to compare, which would render as multiple lines in the chart. The $PlatformShort$ Spline chart is identical to the $PlatformShort$ Spline chart, the only different being the line chart is points connected by straight lines, and the spline chart points are connected by smooth curves. Learn more about our [spline charts](chart-types-spline.md).
+
+### Stepline Chart 
+
+A Step Line Chart, or Step Line Graph, is a category charts that renders a collection of data points connected by continuous vertical and horizontal lines forming a step-like progression. Values are represented on the Y-Axis (left labels) and categories are displayed on the X-Axis (bottom labels). The $PlatformShort$ Step Line chart emphasizes the amount of change over a period of time or compares multiple items. The $PlatformShort$ Step Line chart is identical to the $PlatformShort$ step area chart in all aspects except that the area below the step lines is not filled in. Learn more about our [step line charts](chart-types-step.md)
+
+### Timeline Charts / Time-Series Charts 
+
```

Comment:
Is that the type of chart that have to be designed? I am giving basic examples.
![Timeline-Chart-in-Excel](https://user-images.githubusercontent.com/57216334/106158915-eaf25d80-618c-11eb-9a93-0ec1ec033845.png)

Or like this one (Gantt)?
https://www.thinkoutsidetheslide.com/wp-content/uploads/2017/11/GanttMonthlyWithMilestones.jpg